index.js

```diff
@@ -116,7 +116,13 @@ module.exports = (input, opts) => {
 					}
 				},
 				task: () => {
-					const args = ['publish', '--new-version', input];
+					const args = ['publish'];
+
+					if (opts.contents) {
+						args.push(opts.contents);
```

Comment:
Should we keep `const args = ['publish', '--new-version', input]` and `splice` it in at position 1 when `opts.contents` is set?

Reply:
This is related to the same approach that exists on `publish` function. It's more legible and describes more what's happening on the code